packages/core/components/FileDetails/index.tsx

```diff
@@ -99,7 +99,8 @@ export default function FileDetails(props: Props) {
             if (isOnWeb && isZarr) {
```

Comment:
I know this isn't part of this work - but why can't we create a thumbnail for desktop users? @BrianWhitneyAI is this what your other comment is about?

Reply:
It's honestly been a while since I've looked into this, but I think that reading a local file from the web requires lots of subfile access. Id have to double check the dependency and test if for thumbnails to verify but knowing how zarrs are organized I would guess we have the same problem as the size request.